feature-libs/asm/asm-customer-360/sections/asm-customer-product-reviews/asm-customer-product-reviews.component.spec.ts

```diff
@@ -108,7 +115,7 @@ describe('AsmCustomerProductReviewsComponent', () => {
         rating: '4',
         reviewStatus: 'Complete',
         reviewText: '',
-        item: 'Shoes, SKU: shoes',
+        item: 'Shoes, asm.customer360.productReviews.sku shoes',
```

Comment:
We probably want the colon outside the translation string. 
```suggestion
        item: 'Shoes, asm.customer360.productReviews.sku: shoes',
```